File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int length = 2 * nums.length;
        int[] ans = new int[length];
        for (int i = 0; i < length / 2; i++) {
            ans[i] = nums[i];
        }
        for (int i = length / 2, j = 0; i < length; i++, j++) {
            ans[i] = nums[j];
        }
        return ans;
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        count = {}
        for i in range(len(nums)):
            count[nums[i]] = 1 + count.get(nums[i], 0)
        
        return max(count, key=lambda x: count[x])
